lib/streamlit/cli.py

```diff
@@ -96,12 +98,32 @@ def main_hello():
 
 
 @main.command('run')
-@click.argument('file', type=click.Path(exists=True))
```

Comment:
moved this validation in the function... I tried fancier options, like having validation callback and thought of a custom `ModuleOrFile` type, but it is "hard" to capture the context (whether `-m` is set or not) at this level.